cmd/livepeer/livepeer.go

```diff
@@ -101,7 +101,7 @@ func main() {
 	verbosity := flag.String("v", "", "Log verbosity.  {4|5|6}")
 	faceValue := flag.Float64("faceValue", 0, "The faceValue to expect in PM tickets, denominated in ETH (e.g. 0.3)")
 	winProb := flag.Float64("winProb", 0, "The win probability to expect in PM tickets, as a percent float between 0 and 100 (e.g. 5.3)")
-	maxSessions := flag.Int("maxSessions", 10, "Orchestrator only. Maximum number of concurrent transcoding sessions")
+	maxSessions := flag.Int("maxSessions", 0, "Maximum number of concurrent transcoding sessions for Orchestrator (default is 10) or maximum number or RTMP streams for Broadcaster (not set by default)")
```

Comment:
Does it seem reasonable to have a default limit for B? We could keep the cap for 10 Bs, which would simplify a number of the other default cases here. If a B wants unlimited sessions it can just set `-maxSessions 0`.

Reply:
Good that you've pointed that out - right now we have different behavior for O and B - zero for B means unlimited, but for O it really means zero - this is not very intuitive.
I think we should make it same - say, default is 10 for both, but zero is unlimited for O too.
wdyt?

Reply:
Unlimited O seems fraught. I'm OK with the minor inconsistency, or perhaps we should just not allow unlimited sessions for B?

Reply:
Set default 10 to B and zero as unlimited for B